plugins/plugin-languageserver/che-plugin-languageserver-ide/pom.xml

```diff
@@ -116,9 +116,20 @@
             <groupId>org.eclipse.lsp4j</groupId>
             <artifactId>org.eclipse.lsp4j.jsonrpc</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.eclipse.xtend</groupId>
+            <artifactId>org.eclipse.xtend.lib</artifactId>
+            <version>${org.eclipse.xtext.version}</version>
```

Comment:
you don't need to declare version here. It's already declared in this pr https://github.com/eclipse/che-parent/pulls

Reply:
Yeah, I just wanted to be explicit with the fact that this is required. But you're right, it can be inherited from parent pom.

Reply:
would you like me to move the dep on org.eclipse.xtext.xbase.lib.gwt to the parent pom too?

Reply:
Yes. Remove it please